src/chapters/angular-lsn3/studio.rst

```diff
@@ -0,0 +1,200 @@
+Studio: Angular, Part 3
+========================
+
+In this studio, you must combine data-binding and attribute directives to
+dynamically change the appearance or behavior of HTML elements.
+
+Getting Started
+----------------
+
+This studio uses the same mission planner repository as Angular studio parts 1
+and 2.
+
+#. Open the `mission planner repository <https://github.com/LaunchCodeEducation/angular-lc101-mission-planner>`__
```

Comment:
this should probably link directly to studio-3 branch, no?

https://github.com/LaunchCodeEducation/angular-lc101-mission-planner/tree/studio-3

Reply:
Updated the link, but by this time the students should have already forked and cloned the repository! :-)